build/azure-pipelines/sql-product-build.yml

```diff
@@ -106,17 +106,28 @@ stages:
         timeoutInMinutes: 90
 
   - stage: Windows
-    condition: and(succeeded(), eq(variables['VSCODE_BUILD_WIN32'], true))
+    condition: and(succeeded(), or(eq(variables['VSCODE_BUILD_WIN32'], true), eq(variables['VSCODE_BUILD_WIN32_ARM64'], true)))
     pool:
       vmImage: 'windows-2019'
     dependsOn:
       - Compile
     jobs:
-      - job: Windows
+      - job: Windows_x64
+        variables:
+          VSCODE_ARCH: x64
         steps:
         - template: win32/sql-product-build-win32.yml
+        condition: and(succeeded(), eq(variables['VSCODE_BUILD_WIN32'], 'true'))
         timeoutInMinutes: 90
 
+      - job: Windows_ARM64
+        variables:
+          VSCODE_ARCH: arm64
+          RUN_TESTS: false # Do not run tests for arm64 build
```

Comment:
Why not?

Reply:
already run tests in the x64 job, no additional coverage added.

Reply:
I think we ideally should be doing tests on ALL architectures - who knows if some library has an issue on some specific arch. Does turning them on hurt/change anything?

At the very least could you come back in and update the comment to explain that? This comment is pretty useless right now - context without having to dig into PR histories is important for stuff like this